### PR TITLE
Final Push: Implement NearbyList Feature, Add Directions, ETA, Distance to Food Truck Detail Screen, Minor UI Changes, Organize, Fix Warnings and Errors

### PR DESCRIPTION
**Pull request by stanleyliu15**

### CODE REVIEW

**Comment by BrandonSYu** [comment]
Everything looks great, and ready for final deployment. Good work overall guys! 👍

**Comment by markify** [comment]
Everything is tested and working, good job team

File: Find Food Trucks SF/Find Food Trucks SF/FoodTruckDetailViewController.swift [hunk 1]
```diff
@@ -181,13 +183,15 @@ class FoodTruckDetailViewController: UIViewController, UITableViewDataSource, UI
             }
             
             let route = response.routes[0]
+            let distance = route.distance.magnitude.rounded()
```
Comment (markify) [inline Find Food Trucks SF/Find Food Trucks SF/FoodTruckDetailViewController.swift]: Distance and routes looks good and calculate the correct miles.

**Review by markify — COMMENTED** [review]
All routes, segues, displaying data, tab transition, loading, maps render, and distance calculate are all tested and working. Build runs and its smooth. Overall great job team.

**Pull request merged**